.travis.yml

```diff
@@ -5,14 +5,16 @@ matrix:
   include:
     - os: linux
       python: 3.6
-      env: PYTHON=3.6
+      env: PYTHON=3.6 OS=LINUX
+      before_script: pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
     - os: osx
       osx_image: xcode9.4
       language: generic
       env: PYTHON=3.6 PYTHONWARNINGS=ignore OS=MAC
```

Comment:
Just curious, why do the mac tests not need the nightly wheels?

Reply:
ah because it looks like the nightly ray wheels aren't supported on this version of osx.